ts/components/services/ContactPreferencesToggles/__test__/ContactPreferencesToggles.test.tsx

```diff
@@ -60,9 +60,9 @@ describe("ContactPreferencesToggles component", () => {
       expect(
         component.getByTestId("contact-preferences-webhook-switch")
       ).toBeDefined();
-      expect(
-        component.getByTestId("contact-preferences-email-switch")
-      ).toBeDefined();
+      // expect(
+      //   component.getByTestId("contact-preferences-email-switch")
+      // ).toBeDefined();
```

Comment:
We comment them out because we will eventually bring them back once [we have full BE support](https://github.com/pagopa/io-app/pull/3187/files#diff-3486218843a18ac6a03c1d9fe9dcd5ff4ad97f8c57f245eb903b088bec911c01R172)?
It makes sense to leave a `TODO` to make it more probable that someone will act on this.

Reply:
Yes, we will reintroduce both the component and tests once the backend will support again the email optin for single services. Good point anyway, I'll add TODOs on any point related to this edit